docs/types/builtin-function.md

```diff
@@ -212,6 +212,60 @@ Halts the process for as many `ms` you specified:
 sleep(1000) # sleeps for 1 second
 ```
 
+### source(fileName) -- aka src(), include(), import(), or require() 
```

Comment:
ok maybe having 5 different functions doing the same thing is a bit too much, my bad for the suggestion :-P

Can we limit it to:

* `source` (bash-style)
* `require` (more elegant, used by both ruby, php and NodeJS)

BTW I have plans for `import` ;-)

Reply:
Done